source/wp-content/themes/wporg-news-2021/block-template-parts/front-page/latest-posts.html

```diff
@@ -1,8 +1,8 @@
 <!-- `perPage` must match the value in `offset_paged_posts()`. -->
 <!-- wp:query {"query":{"perPage":5},"tagName":"section","className":"front__latest-posts","align":"full"} -->
 <section class="front__latest-posts alignfull">
-	<!-- wp:heading {"level":2,"className":"front__latest-posts-heading"} -->
-	<h2 class="front__latest-posts-heading">Latest Posts</h2>
+	<!-- wp:heading {"level":2,"className":"front__latest-posts-heading front__section-heading"} -->
+	<h2 class="front__latest-posts-heading front__section-heading">Latest Posts</h2>
```

Comment:
Do we need this extra class? I don't see it used in the CSS.

Reply:
Ah, probably not. I added that thinking I would be able to style all the section headings at the same time, but then discovered that each one is slightly different 😕

Reply:
73617f9

Reply:
Can you remove it from the other templates too?